src/OptiHPLCHandler/empower_methodset_method.py

```diff
@@ -45,12 +47,19 @@ def __init__(self, method_definition: Union[dict, list]):
                     f"Multiple method set methods found: {method_definition['results']}"
                 )
             method_definition = method_definition["results"][0]
+        self.method_name = method_definition["methodName"]
```

Comment:
Do we handle if a method has the same name as the method we wish to push?

Reply:
`EmpowerHandler.PostMethodsetMethod` specifies `?overWriteExisting=false` in the address.

Reply:
added to guardrails